src/cards/stats-card.js

```diff
@@ -203,13 +185,22 @@ const renderStatsCard = (stats = {}, options = { hide: [] }) => {
     return measureText(custom_title ? custom_title : i18n.t("statcard.title"));
   };
 
-  const width = hide_rank
+  /*
+    When hide_rank=true, minimum card width is max of 270 and length of title + paddings.
```

Comment:
Maybe change this to:

```text
When hide_rank=true, the minimum card width is 270 px + the title length and padding.
When hide_rank=false, the minimum card_width is 340 px + the icon width (if show_icons=true).
Numbers are picked by looking at existing dimensions on production.
```

The ``max`` in your old description confused me a bit.